pkg/notify/notifier/wechat/msg.go

```diff
@@ -0,0 +1,296 @@
+package wechat
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	jsoniter "github.com/json-iterator/go"
+	nmv1alpha1 "github.com/kubesphere/notification-manager/pkg/apis/v1alpha1"
+	"github.com/kubesphere/notification-manager/pkg/notify"
+	notifyconfig "github.com/kubesphere/notification-manager/pkg/notify/config"
+	notifier2 "github.com/kubesphere/notification-manager/pkg/notify/notifier"
+	"github.com/prometheus/alertmanager/config"
+	"github.com/prometheus/alertmanager/template"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+)
+
+const (
+	DefaultSendTimeout = time.Second * 3
+	ToUserMax          = 1000
```

Comment:
Maybe we can change these const's name to be more precise
ToUserMax => ToUserBatchSize
ToPartyMax => ToPartyBatchSize
ToTagMax => ToTagBatchSize